<?xml version="1.0" encoding="UTF-8" ?>

<launch>

  <!-- PHASE 0: Go to pre-docking goal -->
  <node pkg="dafan_docking" type="pre_docking" name="pre_docking" output="screen">
    <rosparam file="$(find dafan_docking)/config/phase0_pre_docking.yaml" command="load" />
  </node>

</launch>
